lib/services/nsStarterKitsPageService.ts

```diff
@@ -22,17 +22,21 @@ export class NsStarterKitsPageService implements INsStarterKitsPageService {
             const displayName = pageTemplate.displayName.toLowerCase();
             const newPageDirectory = util.path.join(appPath, "app");
             const pagesDirectory = util.path.join(__dirname, "../pages");
+            const emptyPackageJson: any = {name: "pageTemplate"};
 
             util.fs.emptyDir(pagesDirectory)
                 .then(() => {
+                    return util.fs.outputJson(util.path.join(pagesDirectory, "package.json"), emptyPackageJson);
+                })
+                .then(() => {
                     return util.pageExists(newPageDirectory, pageName);
                 })
                 .then((pageExists: boolean) => {
                     if (pageExists) {
                         return Promise.reject(new Error(`Page with the name "${pageName}" already exists`));
                     }
 
-                    return this.$nsStarterKitsGitService.clonePageTemplate(displayName, pageTemplate.templateFlavor, pagesDirectory);
+                    return this.$nsStarterKitsNpmService.installPageTemplateFromNpm(displayName, pageTemplate.templateFlavor, pagesDirectory);
```

Comment:
file: 'file:///d%3A/GitHub/nativescript-starter-kits/lib/services/nsStarterKitsPageService.ts'
severity: 'Error'
message: 'Exceeds maximum line length of 120 (max-line-length)'
at: '39,1'
source: 'tslint'